queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE NAME LIKE '%mon';
```

Comment:
- [ ] Kindly ensure your screenshot has this query returning all animals that its name ends in "mon".. Currently this query does not return anything in the screenshot you have provided

Your output
![image](https://user-images.githubusercontent.com/81932478/235504862-6dfd2a9d-7ea9-4dac-90a6-0d91c52de0fb.png)